### DISCUSSION TITLE
Is there any way to disable focus outline around the elements for all elements?

### DISCUSSION THREAD

**Discussion by radiorambo** [carbon-design-system/carbon-components-svelte#1448] [Q&A] [answered]
That blue highlighter that appears when ever we click or select some thing?

Thanks

**Comment by metonym** [comment, marked as answer]
The focus outline exists for accessibility. As such, it's not recommended to remove them entirely.

If using the `all.css` stylesheet, you can customize the `--cds-focus` color.

```css
html {
  --cds-focus: #ccc;
}
```

If you must remove it, the easier way would be to set the color to be `transparent`.
